src/pages/guides/getting-started/azure-pipelines.mdx

```diff
@@ -0,0 +1,135 @@
+export const description =
+  'Deploy to AWS, Google Cloud or Microsoft Azure using Azure DevOps and the Nitric CLI'
+
+### Deployment Automation with Azure Pipelines and Nitric
+
+This guide will illustrate how Nitric can be integrated with Azure Pipelines to establish a continuous deployment pipeline. The example below focuses on AWS, but with minor adjustments, it can be adapted for Google Cloud or Microsoft Azure.
```

Comment:
Guide looks good, only thought is that it's a bit strange for us to do an AWS example with Azure Devops.

Reply:
Yeh fair call, will add steps for Azure as the primary.